.github/scripts/build_publish_nightly_docker.sh

```diff
@@ -3,7 +3,7 @@
 set -xeuo pipefail
 
 PYTORCH_DOCKER_TAG=$(git describe --tags --always)-devel
-CUDA_VERSION=11.3.1
+CUDA_VERSION=11.6.1
```

Comment:
11.6.1 or 11.6?

Reply:
It gets docker image from Nvidia docker.io and there we have 11.6.1 but in our scripts we need to use 11.6

Reply:
Not sure I understand the answer, we don't need 11.6, we should use latest minor revision, if we can